app/src/main/java/com/esri/arcgismaps/kotlin/sampleviewer/ui/screens/home/HomeCategoryScreen.kt

```diff
@@ -116,10 +119,25 @@ fun HomeCategoryScreen(
                     CategoryCard(category) { onNavigateToCategory(category.title) }
                 }
             }
+            CheckAccessToken()
         }
     }
 }
 
+@Composable
+fun CheckAccessToken() {
+    var isDialogDisplayed by remember {
+        mutableStateOf(BuildConfig.ACCESS_TOKEN == "DEFAULT_ACCESS_TOKEN")
+    }
+    if (isDialogDisplayed) {
+        MessageDialog(
+            title = "Using default access token",
```

Comment:
```suggestion
            title = "Access token",
```

Might look better on one line